functions/common/aws-s3-common/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.autoconfigure.exclude=org.springframework.cloud.aws.autoconfigure.context.ContextResourceLoaderAutoConfiguration
```

Comment:
Consider to configure your IDE to always insert a new line in the end of all the files.